bower.json

```diff
@@ -27,18 +27,15 @@
     "rjs.noop.js"
   ],
   "dependencies": {
-    "requirejs": "2.1.17",
-    "requirejs-plugins": "1.0.3",
-    "require-css": "0.1.8",
-    "c3": "0.4.9",
-    "colorbrewer": "1.0.0",
-    "d3": "3.5.0",
-    "dagre": "0.7.1",
```

Comment:
You may want this version of dagre?  As I mentioned before, the bower dagre is a bit broken, so I have included the "pre-built" version in "third_party" which makes my life a lot easier - we can include it here in bower as well?